pkg/api/const.go

```diff
@@ -29,7 +29,7 @@ const (
 	Ubuntu1804        Distro = "ubuntu-18.04"
 	Ubuntu1804Gen2    Distro = "ubuntu-18.04-gen2"
 	RHEL              Distro = "rhel"
-	CoreOS            Distro = "coreos"
+	CoreOS            Distro = "coreos"            // deprecated
```

Comment:
Does this constant remain as a notice of its deprecation, or because code still refers to it somewhere?